bin/rename_project

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
as @olivierobert prefer to use Makefile instead of `bin/*` on my PR, should we made this with Makefile too 🤔 https://github.com/nimblehq/elixir-templates/pull/9#pullrequestreview-447245407

Reply:
Makefile all the things ⚔️

Reply:
@anduonghien @olivierobert 
I tried to look into the Makefile approach, but I think we will end up writing shell script inside the Makefile 😆 (Maybe because I am not very familiar writing Makefile). 

But, I think for this we can go with bash script because this is just a bootstrapping script i.e. IMO, it is not going to be part of a project itself it's just a part of the template. We can make this to remove itself after executing it. Also, I think we can move this script out of the bin(We can avoid introducing `/bin` to project created with this template). 

What do you think?

Reply:
@Tolsee I think it's ok to write the shell script in the Makefile, like @rosle did in the Rails template

![image](https://user-images.githubusercontent.com/11751745/90863059-974b3f00-e3b8-11ea-971c-80ab6b36d60f.png)

Reply:
Also to make it easy for delete, we could name this as `init_app` and then we could delete that file after the job is done.